### PR TITLE
📝: Fix nodejs and ruby versions used in Learn Contents

### LINKED ISSUE #279: M1 MacでPodsのインストール中にエラーが発生
M1 Macで環境構築中にエラーが発生。
CocoaPodsの依存性をインストール中にエラー。

次のURLを参考に対応して解決しました。
https://github.com/CocoaPods/CocoaPods/issues/10684#issuecomment-852723356

Issueという訳ではありませんが（M1 Macのサポート状況は不明なため）、参考までに

### PR DESCRIPTION
**Pull request by creature-water-valley**
## ✅ What's done

- [x] Fix nodejs version(12 => 16).
- [x] Fix nodejs active lts link.
- [x] Fix ruby version on m1 mac
- [x] Change pod install from npx to npm script

---

## Other (messages to reviewers, concerns, etc.)

closes #279

### CODE REVIEW

**Review by kozake — APPROVED** [review]

File: website/docs/react-native/learn/getting-started/setting-up-development-environment.mdx [hunk 1]
```diff
@@ -106,11 +106,15 @@ export PATH=$PATH:$ANDROID_SDK_ROOT/emulator
...
 ### Ruby, CocoaPodsのインストール
 
-macOSには、デフォルトでRubyがインストールされているので、`sudo gem install cocoapods`を実行することで[CocoaPods](https://cocoapods.org)をインストールできます。[^2]
+macOSには、デフォルトでRubyがインストールされているので、`sudo gem install cocoapods`を実行することで[CocoaPods](https://cocoapods.org)をインストールできます。
 
 システム全体にBundlerをインストールするのは避けたい場合や、開発に特権ユーザを利用したくない場合はシステムとは別のRubyをインストールし、その中にBundlerをインストールしてください。[asdf-vm](https://asdf-vm.com/)などのツールを利用すると、システムとは別のRubyを簡単にインストールできます。
 
-[^2]:M1アーキテクチャを使用したMacでCocoaPodsをインストールする場合は、React Native公式のドキュメントに記載されている[Note for Mac M1 users](https://reactnative.dev/docs/environment-setup#cocoapods)を参照してください。
+:::danger warning
+M1アーキテクチャを使用したMacでは、Rubyのバージョンが2.7.x未満の場合にCocoaPodsのインストールでエラーが発生します。
```
Comment (in-a-paddy) [inline website/docs/react-native/learn/getting-started/setting-up-development-environment.mdx:114]:
```suggestion
Appleシリコン搭載のMacコンピュータでは、Rubyのバージョンが2.7.x未満の場合にCocoaPodsのインストールでエラーが発生します。
```

File: website/docs/react-native/learn/getting-started/create-project.md [hunk 2]
```diff
@@ -72,5 +72,5 @@ npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
 
 1. `cd <YourAppName>`（`<YourAppName>`は実際に作成したときの値に変更してください）
 2. `npm install --legacy-peer-deps`
-3. （macOSの場合のみ）`npx pod-install`
+3. （macOSの場合のみ）`npm run pod-install`
```
Comment (in-a-paddy) [inline website/docs/react-native/learn/getting-started/create-project.md:75]: rn-spoiler#v2021.12.0 does not contain `pod-install` script. We must remove tag from template uri or revert this change...

Comment (creature-water-valley) [reply website/docs/react-native/learn/getting-started/create-project.md:75]: Change rn-spoiler-tag to master.
ffdf8817912ffc86346c1db412ef93f56d9ea296

**Review by in-a-paddy — APPROVED** [review]
